[#10922] Added top padding to Add All Instructors button

srdeotarse opened the pull request:
Fixes #10922

**PR Checklist**

<!-- Remove this portion after you have made the checks. -->

Ensure that you have:

- [x]  Read and understood our PR guideline: https://github.com/TEAMMATES/teammates/blob/master/docs/process.md#step-4-submit-a-pr

  - [x] Added the issue number to the "Fixes" keyword above
  - [x] Titled the PR as specified in the abovementioned document
- [x] Made your changes on a branch other than `master` and `release`
- [x] Gone through all the changes in this PR and ensured that:
  - [x] They addressed one (and only one) issue
  - [x] No unintended changes were made
- [x] Run and passed static analysis: `./gradlew lint` and `npm run lint`
- [ ] Added/updated tests, if changes in functionality were involved
- [ ] Added/updated documentation to public APIs (classes, methods, variables), if applicable

**Outline of Solution**

<!-- Tell us how you solved the issue. -->
<!-- If there are things you want the reviewers to focus on, include them here as well. -->
<!-- This portion can be skipped if the fix is trivial. -->
Surrounded 'Add All Instructors' button with div containing class="top-padded" in /teammates/src/web/app/pages-admin/admin-home-page/admin-home-page.component.html

Changed top padding of 'Add All Instructors' button form this -
![image](https://user-images.githubusercontent.com/53873549/105030246-8858dd80-5a79-11eb-84c6-fe8cd1b3c058.png)
To this -
![image](https://user-images.githubusercontent.com/53873549/105030299-9ad31700-5a79-11eb-9afb-22c52ea9d40f.png)

Derek-Hardy commented:
There are failures on snapshot test suites because your solution has modified HTML elements in `admin-home-page` component.

Remember to update snapshots before submission. Commonly we do `npm test` followed by option `u` to update failed snapshots after modification.

If unfamiliar, you can refer to the [documentation](https://github.com/TEAMMATES/teammates/blob/master/docs/snapshot-testing.md#how-do-we-use-snapshot-testing) or Jest official website.

Derek-Hardy commented on src/web/app/pages-admin/admin-home-page/admin-home-page.component.html:
```diff
@@ -79,10 +79,12 @@
         </tbody>
       </table>
+      <div class="top-padded">
       <button class="btn btn-primary" (click)="addAllInstructors()" id="add-all-instructors" [disabled]="activeRequests > 0 || isAddingInstructors">
```
It will be great to add indentation here.

srdeotarse replied:
Ok I will add identation and also update snapshots.

srdeotarse commented:
> There are failures on snapshot test suites because your solution has modified HTML elements in `admin-home-page` component.
> 
> Remember to update snapshots before submission. Commonly we do `npm test` followed by option `u` to update failed snapshots after modification.
> 
> If unfamiliar, you can refer to the [documentation](https://github.com/TEAMMATES/teammates/blob/master/docs/snapshot-testing.md#how-do-we-use-snapshot-testing) or Jest official website.

I have removed div with 'top-padded' class, added class 'top-padded' to 'add-all-instructors' button and updated snapshot.

Derek-Hardy reviewed (approved):
LGTM
